CONTRIBUTING.md

```diff
@@ -232,17 +232,17 @@ The following table shows examples of the gentest options.
 
 ### Assertions
 
-- assert_line_num(expected)
+- assert_line_num(exp)
```

Comment:
I'm not sure if this abbreviation is necessary. `expected` is already a common word while `exp` is a bit ambiguous (can be interpreted as `expression`).

Reply:
`exp` is a popular word in minitest.
http://docs.seattlerb.org/minitest/Minitest/Assertions.html

Reply:
I understand what you are saying. I'll revert it.